src/test/unit/inboundApi/handlers.test.js

```diff
@@ -63,15 +67,15 @@ describe('Inbound API handlers:', () => {
             expect(quoteRequestSpy.mock.calls[0][1]).toBe(mockContext.request.headers['fspiop-source']);
```

Comment:
is possible to use `toHaveBeenCalledWith` instead mock.calls[...][...] ?

Reply:
Fixed it